### ISSUE TITLE
As a member of the target audience, e.g a concert manager, instructions can be hard to follow

### ISSUE THREAD

**Issue by TerenceWongsExtravaganza** [TerenceWongsExtravaganza/pe#7]
The instructions in user guide, especially at the quick start section, can be hard for someone of the target audience who may have little technical knowledge to follow.

![10577662cd7d00d3f248b713e7e83a25.png](https://raw.githubusercontent.com/TerenceWongsExtravaganza/pe/main/files/0280243a-a34f-4fde-b97a-cf625477b163.png)

For example, the reader may not understand how to check Java version, or how to cd into a folder, making it impossible for them to use the application.

Since the technical knowledge of target audience is not accounted for and may lead to unusable product, it is medium severity.

**Comment by nus-pe-script** [comment]
# Team's Response
Hi thanks for the feedback. We understand that some may not have the technical knowledge required to use the product. Foreseeing this issue, we have stated the target user profile is reasonably comfortable using CLI apps. So, if the person does not possess the technical knowledge then he will be deemed as not part of the target user profile. As such, checking the java version using `--version` is actually a common flag to be used for CLI apps and the use of `cd` is a valid assumption under the target user we have identified.

![Screenshot 2024-11-18 at 12.09.57 AM.png](https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/a16a1724-838f-4cd8-8ce0-35a5fc44ebd2.png)
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.Rejected`]

- [x] I disagree

**Reason for disagreement:** I disagree as being comfortable with CLI application doesnt make a user know all the different commands and stuff. What if the kind of CLI app the user uses doesnt require any kind of OS specific commands to use? 

<catcher-end-of-segment><hr>
## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.Medium`]

- [x] I disagree

**Reason for disagreement:** I disagree as it can cause quite an inconvenience to users if they do not know the different steps which can maybe even end of them not using the application at all

<catcher-end-of-segment><hr>
